### ISSUE TITLE
How can I add other allowed document upload types so TD can test imports?

### ISSUE THREAD

**Issue by tonypesdm** [tonypesdm/CAMS#76]

**Comment by joerichardsESDM** [comment]
Yes, there is a filter based on a regex matching the file extension in two places: the 1st which is easy to change is in FileUploadControlPage.aspx on line 176. The 2nd is within the FileHandler.ashx compiled code and so can't be changed without recompiling the DLL. I would suggest the best course of action is to create a web.config setting for this regex and then change the code in those two places to pick the regex up from that setting. This will make it more configurable and prevent the two regex's getting out of synch.

**Comment by joerichardsESDM** [comment]
Also, note previous discussion of a related issue in #37

**Comment by joerichardsESDM** [comment]
Single config <1 hour. The other stuff, off the top of my head, 4 - 6 hours depending if there are any traps in there.

**Comment by tonypesdm** [comment]
Limit to 2hrs in first instance, then report back.

**Comment by joerichardsESDM** [comment]
I've made the change to make the list of accepted file types configurable. I've also looked over the handler code and identified key sections wher the code is expecting image files and which will need to be rewritten and/or bypassed with new code. Estimate 4hrs to do that.
